StepsToCare/CareDashboard.ascx.cs

```diff
@@ -82,6 +82,20 @@ namespace RockWeb.Plugins.rocks_kfs.StepsToCare
         Order = 4,
         Key = AttributeKey.MinimumCareTouchHours )]
 
+    [IntegerField(
+        "Minimum Follow Up Care Touch Hours",
+        Description = "Minimum hours for the follow up worker to add a care touch before the need gets 'flagged'.",
+        DefaultIntegerValue = 24,
+        IsRequired = true,
+        Order = 4,
+        Key = AttributeKey.MinimumFollowUpTouchHours )]
+
+    [BooleanField( "Enter Care Need Edit Permission",
+        Description = "Should the \"Enter Care Need\" large button be protected by the edit permission?",
```

Comment:
```suggestion
    [BooleanField( "Override New Care Need Edit Permission",
        Description = "Should the Add Care Need buttons be protected by the edit permission?",
```
Ensure it is the direction you want it to be in the logic and description.